core/src/main/scala/cats/MonadTrans.scala

```diff
@@ -0,0 +1,16 @@
+package cats
+
+/**
+  * A type class which abstracts over the ability to lift an M[A] into a
+  * MonadTransformer
+  */
```

Comment:
nit here: the scala doc style in cats is 
```
/**
 *
 */
``` 
i.e. all stars aligned with the first star.